app/models/division.rb

```diff
@@ -50,8 +50,7 @@ def self.root_internal_name
 
   def self.root
     # create on demand if not present for convenience of blank db's and test cases
-    # @@root ||=
```

Comment:
I'm not sure *where* in the codebase it makes the most sense to do this yet, but we could `raise` an error in the event a root division cannot be found. Probably, if it's fundamental, a check like this will make it somewhere into `application_controller.rb`, for some things this is a potentially larger issue than others, so I'd like to defer turning this into a global failure condition until we've sorted through whether a single overarching global root division is strictly necessary, which is something I anticipate we'll discover when implementing the authorization system.